core/warning.js

```diff
@@ -144,9 +147,9 @@ Blockly.Warning.prototype.setText = function(text, id) {
  * Get this warning's texts.
  * @return {string} All texts concatenated into one string.
  */
-Blockly.Warning.prototype.getText = function() {
-  var allWarnings = [];
-  for (var id in this.text_) {
+Warning.prototype.getText = function() {
+  const allWarnings = [];
+  for (let id in this.text_) {
```

Comment:
I'm not sure what the consensus is for this, but I think it was leaning towards `const`.

Reply:
I've checked, consensus is `const` I will be updating.